wiki/Tournaments/MCNC/2020/en.md

```diff
@@ -0,0 +1,406 @@
+---
+tags:
+  - MCNC
```

Comment:
```suggestion
  - MCNC 2020
```

Reply:
MCNC 2020 is in Line 4

Reply:
tags are there to help the search engine categorise better. it values tags much higher than text so adding this is worthwhile

Reply:
I mean, there are 2 tags in the article, MCNC and MCNC 2020. The tag MCNC 2020 is in Line 4

Reply:
`MCNC2020` and `MCNC 2020` are two different tags

Reply:
the tags here should be `MCNC`, and `MCNC2020`. a search for "MCNC 2020" will match the MCNC tag and 2020 from the title, you don't need that in tags